src/constants/PropTypes.js

```diff
@@ -116,7 +116,8 @@ export const TableCardPropTypes = {
         width: PropTypes.number,
         label: PropTypes.string.isRequired,
         priority: PropTypes.number,
-        renderer: PropTypes.func,
```

Comment:
Is this prop something that worked at all? So, is this a breaking change here? If the feature is completely broken in its current state then we can probably safely remove this prop but if there are possible users out there we will need to add a deprecation warning and alias it to the correct prop name.

Reply:
From what I could see this did not work at all, it was simply passed down to the Card component where it was ignored. Not sure about if it ever worked in the past.

Reply:
Went back as far as commit 9e3490b7 on 5 June 2019 when the `TableCardPropTypes` were added to the `PropTypes.js` file but even then the 'renderer' did not work. Replacing the `renderer` with `renderDataFunction` also did the trick back then though.

Reply:
Great! In that case we can forego a breaking change.